petermwash/Top-News
# feat: Network Unitility class

#### What does this PR do?
- Implement NetworkUtility class.

#### Description of Task to be completed?
- [x] Implement a NetworkUtility class.
- [x] Inform the user in case of unavailable network

#### How should this be manually tested?
- Fetch this branch `git fetch origin ft-network-utility-class-167382898`
- Checkout to the branch then run the app on an emulator or a real device.
- Put your device/emulator in flight mode then click on the `everything` or `top headlines` tab to make a request to the API.
- The app should show a Snackbar at the bottom of the screen informing you of network unavailable.

#### Screenshot:
![networkutility](https://user-images.githubusercontent.com/27728464/61800644-11b43300-ae36-11e9-995c-9b6cd6a1b239.gif)

#### What are the relevant pivotal tracker stories?
[#167382898](https://www.pivotaltracker.com/story/show/167382898)
